nevergrad/functions/iohprofiler/test_core.py

```diff
@@ -7,18 +7,31 @@
 import pytest
 import numpy as np
 from . import core
+from nevergrad.optimization.optimizerlib import OnePlusOne
 
 
 @pytest.mark.parametrize("fid", range(1, 24))  # type: ignore
 def test_PBO(fid: int) -> None:
-    try:
-        func = core.PBOFunction(fid, 0, 16)
-    except ModuleNotFoundError:
-        raise SkipTest("IOH is not installed")
-    x = func.parametrization.sample()
-    value = func(x.value)
-    assert isinstance(value, float), "All output of the iohprofiler-functions should be float"
-    assert np.isfinite(value)
+    for iid in range(1, 5):
```

Comment:
use pytest.mark.parametrize for iid as well

Reply:
done